doc/Extensions/Distributed-Poller.md

```diff
@@ -41,23 +41,26 @@ These requirements are above the normal requirements for a full LibreNMS install
 By default, all hosts are shared and have the `poller_group = 0`. To
 pin a device to a poller, set it to a value greater than 0 and set the
 same value in the poller's config with
-`$config['distributed_poller_group']`. One can also specify a comma
+`distributed_poller_group`. One can also specify a comma
 separated string of poller groups in
-$config['distributed_poller_group'].  The poller will then poll
+`distributed_poller_group`.  The poller will then poll
 devices from any of the groups listed.  If new devices get added from
 the poller they will be assigned to the first poller group in the list
 unless the group is specified when adding the device.
 
 The following is a standard config, combined with a locking mechanism below:
 
-```php
-// Distributed Poller-Settings
-$config['distributed_poller']                            = true;
-// optional: defaults to hostname
-#$config['distributed_poller_name']                      = 'custom';
-$config['distributed_poller_group']                      = 0;
+!!! setting "poller/distributed"
+    ```bash
+    lnms config:set distributed_poller true
+    lnms config:set distributed_poller_group 0
+    ```
+
+If you want to customise the hostname for the poller then you will need
+to set this in `config.php`:
+
+`lnms config:set distributed_poller_name 'custom'`
```

Comment:
This one actually needs to be set in config.php otherwise it applies to all nodes.

Reply:
it should go in .env in that case?

Reply:
yes, and there is an .env for the dispatcher, but not the python wrapper.

Maybe it is time to deprecate using the python wrapper for distributed polling?

Reply:
Well spotted, I've reverted that change. Didn't change it elsewhere, must have just got over zealous with this one!